specificatie/openapi.yaml

```diff
@@ -1300,6 +1321,8 @@ components:
       type: "object"
       description: |
                     Gegevens over het gezag over de persoon.
+                    * *indicatieGezagMinderjarige** wordt gevuld op basis van de waarden die voorkomen in de tabel 'indicatiegezagminderjarige' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
Gezien de opmerking van @fsamwel ('De huidige landelijke tabellen hebben een code met onderscores') vraag ik me af of de tabelnaam 'indicatiegezagminderjarige' wel de juiste naam is.

Reply:
Is gezag_minderjarige geworden